mobile/src/screens/components/Notation.ts

```diff
@@ -0,0 +1,37 @@
+// Initial chessboard code credits go to William Candillon
+// Github: https://github.com/wcandillon
+// Source Code: https://github.com/wcandillon/can-it-be-done-in-react-native/tree/master/season4/src/Chess
+import { Position } from "chess.js";
+import { Dimensions } from "react-native";
+import { Vector } from "react-native-redash";
+
+const { width } = Dimensions.get("window");
+export const SIZE = width / 8;
+
+export const toTranslation = (to: Position) => {
+  "worklet";
```

Comment:
What does this line do? Line 32 is similar.

Reply:
Basically worklets are pieces of JS code from the react native reanimated library. At a high level, the worklet enable smooth animations when dragging the chess piece around.